apps/src/templates/lessonOverview/lessonPlanShapes.jsx

```diff
@@ -1,5 +1,70 @@
 import PropTypes from 'prop-types';
 
+export const levelShape = PropTypes.shape({
+  // id of level
+  id: PropTypes.number.isRequired,
+  //name of level
+  name: PropTypes.string.isRequired,
+
+  //information used to preview levels in activity preview
+  icon: PropTypes.string,
+  isUnplugged: PropTypes.bool,
+  isConceptLevel: PropTypes.bool,
+
+  // blockly options
+  conceptDifficulty: PropTypes.string,
+  concepts: PropTypes.string,
+  skin: PropTypes.string,
+  videoKey: PropTypes.string
+});
+
+export const scriptLevelShape = PropTypes.shape({
```

Comment:
heads up @cforkish that this adds one more place that will need to be updated in your PR which affects level ids.